src/components/Dashboard/FilingHistoryList/bodies/FutureEffective.vue

```diff
@@ -69,6 +69,13 @@ export default class FutureEffective extends Vue {
         filingTitle: 'Voluntary Dissolution'
       }
     }
+    if (EnumUtilities.isTypeContinuationIn(this.filing)) {
+      return {
+        subtitle: 'Future Effective Incorporation Date',
+        filingLabel: 'continuation in',
+        filingTitle: 'Continuation In Application'
+      }
+    }
```

Comment:
I added this case for non-default subtitle/label/title for this filing type.

Yes, the subtitle refers to an "Incorporation", same as the UI design for a completed continuation in:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/005520fc-f1a2-4cb8-9f58-bc49a3e63ef4)